config/source/multi/deployments/adapter.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   labels:
     kafka.eventing.knative.dev/release: devel
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
not sure this is correct, to set that on default.
should be more done via an operator thing, deploying the bits